packages/dashboard/src/pages/Settings/components/LanguageSetting.tsx

```diff
@@ -0,0 +1,20 @@
+import { MenuItem } from '@material-ui/core'
+import { useLanguage } from '../api'
+import { Services } from '../../../API'
+import SettingSelect from './SettingSelect'
+
+export default function LanguageSetting() {
+    let lang = useLanguage()
+    const handleChange = (event: any) => {
+        Services.Settings.setLanguage(event.target.value)
+    }
+
+    return (
+        <SettingSelect value={lang} onChange={handleChange}>
+            <MenuItem value="zh">中文</MenuItem>
+            <MenuItem value="en">English</MenuItem>
+            <MenuItem value="ko">한국인</MenuItem>
+            <MenuItem value="ja">日本語</MenuItem>
```

Comment:
The same as above. And enum `Language` should be moved to the `theme` package or `shared` IMO.